bin/vagrant_provision.sh

```diff
@@ -69,7 +69,6 @@ VENV=/home/vagrant/virtualenv
 sudo -H -u vagrant -s -- <<EOF
 virtualenv $VENV
 source $VENV/bin/activate
-pip install -U "pip<7"
```

Comment:
Hmm. The vagrant box maybe Ubuntu 14.04. It has pip 1.5.x. So I think this should be upgrade to a fixed version of ``7.1.2`` which peep supports.